### PR TITLE
Migrate docs 

### PR DESCRIPTION
**Pull request by hectorakemp**
### What type of PR is this?

_(documentation)_

### What this PR does / Why we need it?
This pull request adapts the documentation from the internal backplane repository for a public audience. It adds the old README and the guide for setting up the bash/zsh PS1. 

### Which Jira/Github issue(s) does this PR fix?

_Resolves #_ [OSD-15318](https://issues.redhat.com/browse/OSD-15318)

### Pre-checks (if applicable)

- [ ] Ran unit tests locally
- [x] Validated the changes in a cluster
- [x] Included documentation changes with PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,331 @@
-# Backplane-CLI
+# backplane-cli
 
-## What it does
-Backplane-cli is a CLI tool to interact with Backplane-api, which is a management service that consolidates all access to OSD and ROSA clusters and cloud provider resources. Backplane handles authentication and authorization through OCM profiles, practices least privilege access to those OSD/ROSA clusters and cloud resources, and provides auditing capabilities for compliance and security.
+backplane-cli is a CLI tool to interact with backplane api.
+
+The onboarding documentation can be found on [The Source Wiki](https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_user_documentation).
```
Comment (wanghaoran1988) [inline README.md:5]: Let's remove this for now.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,331 @@
...
+
+## Install
+
+### Option 1: Use `go install`
+
+You need to have `go` installed, if you want to use this method,
+the minimal version required is `go 1.16`. You can check this with `go version`
```
Comment (wanghaoran1988) [inline README.md:12]: Should be go 1.18 ? https://github.com/openshift/backplane-cli/blob/main/go.mod#L3

File: README.md [hunk 3]
```diff
@@ -1,4 +1,331 @@
...
+
+Then run `GOPRIVATE=github.com go install github.com/openshift/backplane-cli/cmd/ocm-backplane@{TAG_VERSION}`
```
Comment (wanghaoran1988) [inline README.md:14]: Do we need GOPRIVATE for github?

File: README.md [hunk 4]
```diff
@@ -1,4 +1,331 @@
...
+| `ocm-backplane testJob get <job_name> [flags]`                              | Retrieve a backplane test job resource                                                   |
+| `ocm-backplane testJob list [flags]`                                        | Retrieve a list of backplane test job resources                                          |
+| `ocm-backplane testJob logs <job_name> [flags]`                             | Retrieve logs of the specified test job resource                                         |
+| `ocm-backplane upgrade`                                                     | Upgrade backplane-cli to the latest version                                              |
+| `ocm-backplane version`                                                     | Display the installed backplane-cli version                                              |
+
+### Example
+
+In this example, we will login to a cluster with id `123456abcdef` in production environment, and we have the OCM client environment setup [like this](https://github.com/openshift/ops-sop/blob/master/v4/knowledge_base/ocm.md#prod-and-staging-at-the-same-time).
```
Comment (wanghaoran1988) [inline README.md:135]: Let's use https://github.com/openshift-online/ocm-cli#log-in in stead of the private sop repo.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,331 @@
...
+
+### Project
+
+When executing `ocm backplane project <project-name>`, it will:
+
+- Manipulate the `kubeconfig` and set the namespace of the current context to `project-name`.
+
+This serves as a workaround for the [oc command issue](https://github.com/openshift/oc/issues/647).
+
+### Monitoring UI Access
```
Comment (wanghaoran1988) [inline README.md:227]: Let's remove this part, it's been deprecated.

File: README.md [hunk 6]
```diff
@@ -1,4 +1,331 @@
...
+  $ ocm backplane cloud console
+  ```
+
+### Managed Scripts
+
+- Backplane scripts allows backplane users to run pre-defined scripts that require privileged permissions on OSD clusters.
+- It allows SRE/CEE to execute pre-approved, higher privileged scripts without admin access, hence no access elevation alert will be triggered.
+- Each script has limited RBAC permissions, this is done via a metadata file associated with the script.
+- These scripts are stored in a separate repository called [managed-scripts](https://github.com/openshift/managed-scripts)
+- Instructions on how to submit a new managed script is detailed [here](https://source.redhat.com/groups/public/openshiftplatformsre/wiki/backplane_scripts).
```
Comment (wanghaoran1988) [inline README.md:292]: Let's remove this.

File: README.md [hunk 7]
```diff
@@ -1,4 +1,331 @@
...
+  - If not then fetch all jobs, sends a GET request to the backplane service: `GET https://<backplane-url>/backplane/script/{clusterId}/job`
+  - Parses the response and displays managed job(s) running in the cluster.
+- Fetch managed job logs, `ocm backplane managedjob logs <jobID>`:
+  - Sends a GET request to the backplane service: `GET https://<backplane-url>/backplane/{clusterId}/job/{jobId}/logs`
+  - Parses the response and displays the managed job logs.
+- Delete a managed job, `ocm backplane delete <jobName>`:
+  - Stops a running job and deletes all of its resources (including logs).
+  - Sends a DELETE request to thr backplane service: `DELETE https://<backplane-url>/backplane/{clusterId}/job/{jobId}`
+
+## Feedback
```
Comment (wanghaoran1988) [inline README.md:326]: Let's remove this for now?

File: docs/PS1-setup.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
+# How to setup PS1 in bash/zsh
+
+You can use the below methods to set the shell prompt, so you can learn which cluster operating on, like
+~~~
+[siwu@siwu ~ (⎈ |stg/siwu-test-1.zlob.s1:default)]$ date
+Tue Sep  7 17:40:35 CST 2021
+[siwu@siwu ~ (⎈ |stg/siwu-test-1.zlob.s1:default)]$ oc whoami
+system:serviceaccount:openshift-backplane-srep:xxxxxxxxxxxx
+~~~
```
Comment (wanghaoran1988) [inline docs/PS1-setup.md:9]: Let's remove this example? or update to not include the 'siwu' ?

File: README.md [hunk 9]
```diff
@@ -1,4 +1,197 @@
...
+  ```
+  $ ocm-container <cluster-name/cluster-id>
+  ```
+
+  Manual login:
+
+  ```
+  $ ocm-container
+  (in-container)$ backplane-login <cluster-name/cluster-id>
+  ```
```
Comment (supreeth7) [inline README.md:99]: We do not have `ocm-container` scripts for now, so this can be omitted.

**Review by supreeth7 — CHANGES_REQUESTED** [review]

**Pull request merged**